static/src/javascripts-legacy/projects/commercial/modules/dfp/load-advert.js

```diff
@@ -5,8 +5,14 @@ define([
     return loadAdvert;
 
     function loadAdvert(advert) {
-        Advert.startLoading(advert);
-        window.googletag.display(advert.id);
-        dfpEnv.firstAdDisplayed = true;
+        advert.whenSlotReady
```

Comment:
I wanted to, but some of the metrics have been sent as `loadStart` and `loadEnd` for a while now, which made me think twice. Also, our `Advert` object has `rendering` vars, and it's not going to be obvious what the difference is between "display" and "render". The real answer lies in Googletag's implementation, where render occurs within display, and display really means "load and render".